src/test/java/org/broadinstitute/hellbender/utils/GenomeLocParserUnitTest.java

```diff
@@ -504,4 +505,21 @@ public void testGenomeLocPadding(final GenomeLoc input, final int pad) {
         Assert.assertEquals(padded.getStop(), Math.min(input.getStop() + pad, contigLength));
     }
 
+    @Test
+    public void testQueryAllHG38Intervals() {
+        SAMSequenceDictionary sd;
+        final File testFile = new File ("src/test/resources/org/broadinstitute/hellbender/engine/Homo_sapiens_assembly38.headerOnly.vcf.gz");
+
+        try (VCFFileReader vcfReader = new VCFFileReader(testFile, false)) {
+            sd = vcfReader.getFileHeader().getSequenceDictionary();
+        }
+
+        // Test that we can use any contig from hg38 as a query against a VCF with an hg38 sequence dictionary without
+        // having an ambiguity.
+        final GenomeLocParser localGenomeLocParser = new GenomeLocParser(sd);
+        sd.getSequences().stream().forEach(hg38Contig -> localGenomeLocParser.parseGenomeLoc(hg38Contig.getSequenceName()));
+        sd.getSequences().stream().forEach(hg38Contig -> localGenomeLocParser.parseGenomeLoc(hg38Contig.getSequenceName() + ":1-1"));
+        sd.getSequences().stream().forEach(hg38Contig -> localGenomeLocParser.parseGenomeLoc(hg38Contig.getSequenceName() + ":1+"));
```

Comment:
Add a test for the "contig:1" case here. Also, assert that the return value of `parseGenomeLoc()` is as expected

Reply:
Done.